app/controllers/users_controller.rb

```diff
@@ -1,6 +1,15 @@
 class UsersController < ApplicationController
+  before_action :logged_in_user, only: [:index, :edit, :update, :destroy]
```

Comment:
```suggestion
  before_action :logged_in_user, only: %i[index edit update destroy]
```
nits
こちらの方がちょっとだけスッキリ書けます。